### PR TITLE
amass*: edit pages

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fazlearefin** [comment]
`amass` has gone through many changes in the most recent version (**v4.2.0**) and hence the pages needed maintenance updates.

Main change is the removal of `db` subcommand from `amass` to a new tool called [oam-tools](https://github.com/owasp-amass/oam-tools). The commits are here:
- https://github.com/owasp-amass/amass/commit/87f73be3c7121e3771b59ff2120595378c05cdc3
- https://github.com/owasp-amass/amass/commit/8c6361c66dc506fa120cf5f0f0ee544c24724a86

The documentation for the tool in the READMEs also have not reflected this change. In order to find the most accurate help, install the tool and run `amass -h`, `amass enum -h`, and `amass intel -h`

File: pages/common/amass-intel.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-> More information: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-intel-subcommand>.
+> More information: <https://github.com/owasp-amass/amass/blob/master/doc/user_guide.md#the-intel-subcommand>.
 
-- Find root domains in an IP address range:
+- Find root domains in an IP [addr]ess range:
 
 `amass intel -addr {{192.168.0.1-254}}`
 
-- Use active recon methods:
+- Use [active] recon methods:
```
Comment (vitorhcl) [inline pages/common/amass-intel.md:10]:
```suggestion
- Use active recon methods:
```

File: pages/common/amass-intel.md [hunk 2]
```diff
@@ -26,3 +26,7 @@
 - Save results to a text file:
 
 `amass intel -o {{output_file}} -whois -d {{domain_name}}`
+
+- [list] all available data sources:
```
Comment (vitorhcl) [inline pages/common/amass-intel.md:30]:
```suggestion
- List all available data sources:
```

File: pages/common/amass.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+> Some subcommands such as `amass intel` have their own usage documentation.
+> More information: <https://github.com/owasp-amass/amass>.
 
 - Execute an Amass subcommand:
 
-`amass {{subcommand}}`
+`amass {{intel|enum}} {{options}}`
 
-- Display help:
+- Display [help]:
```
Comment (vitorhcl) [inline pages/common/amass.md:11]:
```suggestion
- Display help:
```

File: pages/common/amass.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 `amass -help`
 
-- Display help on an Amass subcommand (like `intel`, `enum`, etc.):
+- Display help on an Amass subcommand:
 
-`amass -help {{subcommand}}`
+`amass {{intel|enum}} -help`
 
-- Display version:
+- Display [version]:
```
Comment (vitorhcl) [inline pages/common/amass.md:19]:
```suggestion
- Display version:
```

**Review by vitorhcl — COMMENTED** [review]
Not that these are long options, and not short options even though they use just a single "-".

I don't think long option mnemonics for full words adds much value, they are more useful when there are abbreviations or something like that.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/amass-enum.md [hunk 5]
```diff
@@ -1,24 +1,28 @@
...
 
-`amass enum -passive -d {{domain_name}}`
+`amass enum -d {{domain_name}}`
 
 - Find subdomains of a [d]omain and actively verify them attempting to resolve the found subdomains:
 
 `amass enum -active -d {{domain_name}} -p {{80,443,8080}}`
 
-- Do a brute force search for subdomains:
+- Do a [brute] force search for sub[d]omains:
```
Comment (vitorhcl) [inline pages/common/amass-enum.md:14]:
```suggestion
- Do a brute force search for sub[d]omains:
```
Do we want to add a mnemonic for this whole this word? Well, the full, descriptive option would be `-brute-force`, but I don't know if we should highlight this whole word.

**Review by vitorhcl — APPROVED** [review]
Thank you, LGTM after we discuss my minor concern.

**Pull request merged**